Loop Status Extension/StatusViewController.swift

```diff
@@ -263,8 +242,8 @@ class StatusViewController: UIViewController, NCWidgetProviding {
                 let numberFormatter = NumberFormatter()
 
                 numberFormatter.numberStyle = .decimal
-                numberFormatter.minimumFractionDigits = 1
-                numberFormatter.maximumFractionDigits = 1
+                numberFormatter.minimumFractionDigits = 2
+                numberFormatter.maximumFractionDigits = 2
```

Comment:
This looks unrelated to this change. Intentional?

Reply:
Yes, I added a note in the PR description that this change is such that the IOB displayed in the app matches that displayed in the widget.

Reply:
Ah, I missed that. :+1: